server-utils.js

```diff
@@ -86,11 +86,13 @@ module.exports = {
     }
   },
 
-  /**
-   * Only to be used when handling unexpected errors.
-   */
   serverError: function(err, req, res) {
     console.error('Server error:\n', err);
-    respond(req, res, 500, 'Server error');
+    if (err.expected) {
+      const status = err.status || 500;
+      respond(req, res, status, err.message);
+    } else {
+      respond(req, res, 500, 'Server error');
+    }
```

Comment:
I really don't like this. The functions aren't well named, but the documentation that you deleted was there for a reason. Use the `error` function is it's maybe expected, maybe not. Use the `serverError` function if it's unexpected.